jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -150,11 +152,16 @@ protected void navigateToPerspective(final String perspectiveId,
                              parameterValue);
         placeManager.goTo(request);
     }
-
+    
     public boolean isUserAuthorizedForPerspective(final String perspectiveId) {
         final ResourceRef resourceRef = new ResourceRef(perspectiveId,
                                                         ActivityResourceType.PERSPECTIVE);
         return authorizationManager.authorize(resourceRef,
                                               identity);
     }
+    
+    public abstract void openErrorView(final String parameterId);
```

Comment:
You can make these two methods non abstract, that way you only override where needed.